test/conformance/source/environment.cpp

```diff
@@ -184,6 +186,7 @@ PlatformEnvironment::parsePlatformOptions(int argc, char **argv) {
 
 DevicesEnvironment *DevicesEnvironment::instance = nullptr;
 
+// The third expected parameter of argv is test_devices_count.
```

Comment:
change e.g. to: `The third (optional) parameter is expected to be 'test_devices_count'.`

Reply:
Done